src/site/content/en/fast/reduce-javascript-payloads-with-code-splitting/index.md

```diff
@@ -23,6 +26,18 @@ significantly. Instead of shipping all the JavaScript to your user as soon as
 the first page of your application is loaded, split your bundle into
 multiple pieces and only send what's necessary at the very beginning.
 
+## Why is code splitting beneficial?
+
+Code splitting is a measure that seeks to minimize startup time. When we ship less JavaScript at startup, we can get applications to be [interactive faster](/tti/) by minimizing main thread work.
+
+When it comes to [Core Web Vitals](/vitals/), reducing JavaScript payloads downloaded at startup will contribute to better [First Input Delay (FID)](/fid/) and [Interaction to Next Paint (INP)](/inp/) times. The reasoning behind this is that, by freeing up the main thread, the application is able to respond to user inputs more quickly by reducing JavaScript parse, compile, and execution-related startup costs.
+
+Depending on your website's architecture&mdash;particularly if your website relies heavily on client-side rendering&mdash;reducing the size of JavaScript payloads responsible for rendering markup may lead to improved [Largest Contentful Paint (LCP)](/lcp/) times in some situations.
```

Comment:
Reducing scripts can also [impact LCP render time](https://web.dev/optimize-lcp/#2-eliminate-element-render-delay) - as @philipwalton recently discovered!

Not sure how best to fit that in here between your first sentence (that applies to all) and your second, which seems more about late discovered/rendered resources.